src/components/typography/typography.module.scss

```diff
@@ -0,0 +1,37 @@
+@font-face {
+  font-family: "Source Sans Pro";
+  src: url("../../assets/font/source-sans-pro-regular.ttf") format("tff");
```

Comment:
just a dumb question:  Does anything break when the resources are located somewhere else?
Is it relative to this file, isn't it?
Just works ™️  ?!